### ISSUE TITLE
I receive "Availability Status: Closed" alerts, how to filter them?

### ISSUE THREAD

**Issue by vallkor** [juftin/camply#245]
I receive "Availability Status: Closed" notifications for the following cmd:

camply  campsites --rec-area 2931 --start-date 2023-06-09 --end-date 2023-06-13 --notifications pushbullet --search-forever --equipment RV 0 --campground 232461

I can see the code checks the status against CAMPSITE_UNAVAILABLE_STRINGS constant list, "Closed" is in this list, but I still get the alerts where Status is "Closed", how to filter them further to make sure I receive alerts only for "Available" sites?

**Comment by juftin** [comment]
Hey @vallkor , The "Closed" logic was just added yesterday on `0.16.2`, can you confirm you're running with the latest version of camply (`0.18.0`)?

```
camply campsites --rec-area 2931 --start-date 2023-06-09 --end-date 2023-06-13 --continuous --equipment RV 0 --campground 232461 --debug
```

**Comment by vallkor** [comment]
Thank you, @juftin , issue has been fixed with 0.18.0 version. I had an older package installed using Pip and didn't know my python is still using it, I updated packages and it is now fixed. Thanks for a great tool!
